.github/workflows/post-merge-survey.yml

```diff
@@ -0,0 +1,24 @@
+name: Post-Merge Survey
+on:
+  pull_request_target:
+    types: [closed]
+jobs:
+  run:
+    if: >-
+      github.event.pull_request.user.login != 'Dusch4593' &&
+      github.event.pull_request.user.login != 'yangc95' &&
+      github.event.pull_request.user.login != 'KTom101' &&
+      github.event.pull_request.user.login != 'SSwiniarki' &&
+      github.event.pull_request.user.login != 'HishamT' &&
+      github.event.pull_request.user.login != 'caupolicandiaz' &&
+      github.event.pull_request.user.login != 'lucwhite'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: wow-actions/auto-comment@v1
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          pullRequestMerged: |
+             👋 @{{ author }}
+             You have contributed to Codecademy Docs, and we would like to know more about you and your experience.
+             Please take a minute to fill out [this four question survey](https://codecademyready.typeform.com/to/F7tcd9gi) to help us better understand Docs contributions and how we can improve it for you and our learners.
```

Comment:
>Docs contributions and how we can improve it...

I'm not super qualified to review copy but I think there's a mismatch in quantity here. Feels like it should read `them` rather than `it`, or even `Docs contributions and how we can improve the experience...`

Reply:
if you disagree I do not really have any problem shipping this though, just thought I'd ask!

Reply:
Fair point and change made! 👍🏻

Reply:
Actual, I like "the experience" better. 😄